src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -21183,8 +21190,7 @@ void Test2<T2>(T2 x2) where T2 : class, new()
                 );
         }
 
-        // PROTOTYPE(NullableReferenceTypes): Report warnings
-        // if `where T : new()` implies T is non-nullable.
+        // `where T : new()` does not imply T is non-nullable.
```

Comment:
Update the design doc?

Reply:
> Update the design doc?

I do not believe there is anything to update

---
In reply to: [218171343](https://github.com/dotnet/roslyn/pull/29947#discussion_r218171343) [](ancestors = 218171343)

Reply:
A note here to document would be useful: [https://github.com/dotnet/roslyn/blob/features%2FNullableReferenceTypes/docs/features/nullable-reference-types.md#type-parameters](https://github.com/dotnet/roslyn/blob/features%2FNullableReferenceTypes/docs/features/nullable-reference-types.md#type-parameters)

---
In reply to: [218172264](https://github.com/dotnet/roslyn/pull/29947#discussion_r218172264) [](ancestors = 218172264,218171343)

Reply:
> A note here to document would be useful: [https://github.com/dotnet/roslyn/blob/features%2FNullableReferenceTypes/docs/features/nullable-reference-types.md#type-parameters](https://github.com/dotnet/roslyn/blob/features%2FNullableReferenceTypes/docs/features/nullable-reference-types.md#type-parameters)

I don't think obvious things and every random questions need to be noted

---
In reply to: [218172966](https://github.com/dotnet/roslyn/pull/29947#discussion_r218172966) [](ancestors = 218172966,218172264,218171343)